### ISSUE TITLE
Set process priority and affinity through startup settings

### ISSUE THREAD

**Issue by ghost** [PrismLauncher/PrismLauncher#252]
### Role

I use the launcher.

### Suggestion

I want it to be possible to set the game process' priority and CPU affinity from the instance's "Settings" tab, as well as in global settings.

### Benefit

So if one gets a performance boost from setting the game's priority to High, they don't have to use the task manager for this every time they launch the game.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

I don't know what simular system exists for this in Linux (but it surely does).

**Comment by ghost** [comment]
This would be very annoying on linux since you need root to set the priority, and a password prompt everytime you launch minecraft would be annoying.

Maybe on windows it could work

**Comment by ghost** [comment]
yeah on windows it could work; on linux it all could be grayed out unless the launcher has the permission to apply those parameters to the process.

**Comment by Trial97** [comment]
Closing for now as this is not currently planned or feasible.
However, this does not mean the idea is permanently rejected.
If any maintainer believes this is still feasible or worth pursuing, feel free to reopen the issue for further discussion.

**Comment by Aex12** [comment]
I had the same need, and since Prism doesnt support this I though a wrapper program could be useful in here.
I just wrote one, in case anyone is interested: https://github.com/Aex12/lassowrap

Only windows is supported. The usage is pretty simple:
- Download the binary from the releases page
- Place it somewhere in your drive, and annotate the location. I placed mine under C:\Games\lassowrap.exe
- Edit your Minecraft instance in Prism > go into settings > open custom commands tab
- Finally on the wrapper command, write this: `C:\Games\lassowrap.exe --priority high --affinity 0x0000FFFF --` 

That is all required to set the affinity and priority of the java process on launch.
To get the proper affinity mask, you can use any calculator available on the internet for that purpose, for example: https://bitsum.com/tools/cpu-affinity-calculator/

I hope this is useful for someone!
